index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>News summary</title>
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+
```

Comment:
something that I have learnt that might be helpful, is to use <div containers> for each part of the section you need to be styling individually.